packages/flask/src/UIDateTimePicker/Calendar/Calendar.tsx

```diff
@@ -68,8 +83,14 @@ const Column = React.memo(function Column({
 
                 if (day.type === 'dayFiller') {
                     return (
-                        // eslint-disable-next-line react/no-array-index-key
-                        <View style={styles.day} key={`${selectedRow}${index}`}>
+                        <View
+                            style={[
+                                styles.day,
+                                { width: daySize, height: daySize, borderRadius: daySize },
```

Comment:
I'm not sure that `dayFiller` must have `borderRadius` as it's invisible anyway 🤔